### PR TITLE
Fix `HeadingPitchRollValues` type

### PR DESCRIPTION
**Pull request by Gu-Miao**
Make all the properties optional.

Fix issue: #10359

### CODE REVIEW

**Comment by cesium-concierge** [comment]
Thanks for the pull request @Gu-Miao!
* :heavy_check_mark: Signed CLA found.
* :grey_question: [CHANGES.md](https://github.com/CesiumGS/cesium/blob/main/CHANGES.md) was not updated.
    * If this change updates the public API in any way, please add a bullet point to `CHANGES.md`.
* :grey_question: Unit tests were not updated.
    * Make sure you've [updated tests](https://github.com/CesiumGS/cesium/tree/main/Documentation/Contributors/TestingGuide) to reflect your changes, added tests for any new code, and ran the code coverage tool.

Reviewers, don't forget to make sure that:

- [ ] Cesium Viewer works.
- [ ] Works in 2D/CV.

**Comment by Gu-Miao** [comment]
This is ready.

File: Source/Scene/Camera.js [hunk 1]
```diff
@@ -42,9 +42,9 @@ import SceneMode from "./SceneMode.js";
  *
  * An orientation given by numeric heading, pitch, and roll
  *
- * @property {number} heading The heading in radians
- * @property {number} pitch The pitch in radians
- * @property {number} roll The roll in meters
+ * @property {number} [heading] The heading in radians
+ * @property {number} [pitch] The pitch in radians
+ * @property {number} [roll] The roll in meters
```
Comment (ggetz) [inline Source/Scene/Camera.js:47]: Good catch that these should be optional. If a property is optional, we should document what the defaults are. 

```suggestion
 * @property {number} [heading=0.0] The heading in radians
 * @property {number} [pitch=-CesiumMath.PI_OVER_TWO] The pitch in radians
 * @property {number} [roll=0.0] The roll in meters
```

**Comment by ggetz** [comment]
Thanks @Gu-Miao! I just have one comment.

**Comment by Gu-Miao** [comment]
@ggetz Default value is added now.

**Comment by ggetz** [comment]
Thanks @Gu-Miao!

**Pull request merged**